front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -35,21 +39,23 @@ const HANDLE_TAB_RESIZE_HEIGHT = 20;
 
 type SimulationResultsProps = {
   collapsedTimetable: boolean;
-  spaceTimeData?: TrainSpaceTimeData[];
   infraId?: number;
-  trainScheduleUsedForProjection?: TrainScheduleResult;
-  trainIdUsedForProjection?: number;
   simulationResults: SimulationResultsData;
+  projectionData:
+    | {
+        trainSchedule: TrainScheduleResult;
+        projectedTrains: TrainSpaceTimeData[];
+        path: PathfindingResultSuccess;
+        pathGeometry?: PathProperties['geometry'];
```

Comment:
(If you made the change in the custom hook)

```suggestion
        pathGeometry: PathProperties['geometry'];
```

Reply:
Done